explorer/app/components/Logo/Logo.tsx

```diff
@@ -22,6 +22,7 @@ export const Logo: React.FC<LogoProps> = ({
         loader={({ src }) => src}
```

Comment:
For my knowledge, what are `loader` and `unoptimized` for?

Reply:
this loader is the same as the loader on the `next.config.js` file, which at moment is configured as `imgix`, although passing it here as a prop will override the loader on `next.config.js`. 
The loader handles some performing optimizations on images, like serving images with the right size, preventing the layout to shift when the image is loaded, lazy loading, etc.
The loader on `next.config.js` should be `undefined` for now, since I don't know how we're going to serve the images yet, but there's a bug there, that's why I'm setting this here and on `next.config.js`

If the loader prop passed to the image component doesn't hangle `width` and `height` you will get a warning on the console, the `unoptimized` is just to say "I ~~think I~~ know why I doing" and prevent that. 
But this will change as soon as we start serving the real images, so I'll check the best loader for it. But I should add a TODO here, so I won't forget.

let me know if there's anything that needs more clarification

Reply:
Got it, thank you for the detail Bruce.